codespell_lib/data/dictionary.txt

```diff
@@ -5324,6 +5324,7 @@ coloum->column
 coloumn->column
 coloumns->columns
 coloums->columns
+colourfull->colourful
```

Comment:
Suggest colourfully too.

Does this exist in the US too, or do they use two l's?

Reply:
colorfull->colorful was already in.

I added the ly variant for all.

https://dictionary.cambridge.org/de/worterbuch/englisch/colorful